requirements.txt

```diff
@@ -3,7 +3,7 @@ aiohttp
 altair>=4.2.0
 fastapi
 ffmpy
-gradio_client>=0.2.7
+gradio_client>=0.2.9
```

Comment:
So that the latest bug fixes (#4886) are brought in and so that `probe_url` used in the CI.